src/test.tsx

```diff
@@ -5,7 +5,7 @@ import reactNodeToString from "./index"
 type Test = [input: React.ReactNode, expectedOutput: string]
 
 describe("react-node-to-string", () => {
-  it("outputs correct string", () => {
+  it("extract correct string", () => {
```

Comment:
I think it should be "extracts" to get the correct grammar sentence: "it extracts correct string"

Reply:
I agree!